app/assets/stylesheets/application.scss

```diff
@@ -14,3 +14,60 @@
  *= require_self
  */
  @import "bootstrap";
+ html,
+ body {
+   height: 100%;
+   background-color: #f5f5f5;
+ }
+
+ .myloginform {
+   align-items: center;
+   padding-top: 40px;
+   padding-bottom: 40px;
+ }
+
+ .myloginform {
+   max-width: 330px;
+   padding: 15px;
+ }
+
+ .myloginform .form-floating:focus-within {
+   z-index: 2;
+ }
+
+
+ .myloginform input[type="email"] {
+   margin-bottom: -1px;
+   border-bottom-right-radius: 0;
+   border-bottom-left-radius: 0;
+ }
+
+ .myloginform input[type="password"] {
+   margin-bottom: -1px;
+   border-top-left-radius: 0;
+   border-top-right-radius: 0;
+ }
+
+
+ .mysignupform {
+   align-items: center;
+   padding-top: 40px;
+   padding-bottom: 40px;
+ }
+
+ .mysignupform {
+   max-width: 330px;
+   padding: 15px;
+ }
+
+ .mysignupform .form-floating:focus-within {
+   z-index: 2;
+ }
+
+ .mysignupform input[type="email"] {
+   margin-bottom: -1px;
+ }
+
+ .mysignupform input[type="password"] {
+   margin-bottom: -1px;
+ }
```

Comment:
i think we do not need this. Just rename `myloginform` to `myform` and use it in both forms

Reply:
done